annotators/DeepPavlovEmotionClassification/README.md

```diff
@@ -1 +1,15 @@
-BERT Base model for emotion classification which learned at the custom dataset(described more precisely in our article) 
+# DeepPavlov Emotion Classification Annotator
+
+## Description
+
+BERT Base model for emotion classification which learned at the custom dataset (described more precisely in (our article)[link here])
+
+## I/O
+
+**Inpunt**
+
+**Output:**

```

Comment:
?

Reply:
same as Factoid: the info on this one is scarce in our repo, made a ping in our slack channel